spring-cloud-aws-autoconfigure/src/test/java/io/awspring/cloud/autoconfigure/dynamodb/DynamoDbAutoConfigurationTest.java

```diff
@@ -61,34 +64,42 @@ void dynamoDBAutoConfigurationIsDisabled() {
 
 	@Test
 	void dynamoDBAutoConfigurationIsEnabled() {
-		this.contextRunner.withPropertyValues("spring.cloud.aws.dynamodb.enabled:true").run(context -> {
-			assertThat(context).hasSingleBean(DynamoDbClient.class);
-			assertThat(context).hasSingleBean(DynamoDbTemplate.class);
-			assertThat(context).hasSingleBean(DynamoDbEnhancedClient.class);
+		this.contextRunner
+				.withPropertyValues("spring.cloud.aws.dynamodb.enabled:true",
+						"spring.cloud.aws.dynamodb.dax.url:dax://something.dax-clusters.us-east-1.amazonaws.com")
+				.run(context -> {
+					assertThat(context).hasSingleBean(DynamoDbClient.class);
+					assertThat(context).hasSingleBean(DynamoDbTemplate.class);
+					assertThat(context).hasSingleBean(DynamoDbEnhancedClient.class);
 
-			ConfiguredAwsClient client = new ConfiguredAwsClient(context.getBean(DynamoDbClient.class));
-			assertThat(client.getEndpoint()).isEqualTo(URI.create("https://dynamodb.eu-west-1.amazonaws.com"));
-		});
+					ClusterDaxClient client = context.getBean(ClusterDaxClient.class);
+					software.amazon.dax.Configuration configuration = getConfiguration(client);
+					assertThat(configuration.url()).isEqualTo("dax://something.dax-clusters.us-east-1.amazonaws.com");
+				});
 	}
 
 	@Test
 	void withCustomEndpoint() {
-		this.contextRunner.withPropertyValues("spring.cloud.aws.dynamodb.endpoint:http://localhost:8090")
+		this.contextRunner
+				.withPropertyValues(
+						"spring.cloud.aws.dynamodb.dax.url:dax://something.dax-clusters.us-east-1.amazonaws.com")
 				.run(context -> {
 					assertThat(context).hasSingleBean(DynamoDbClient.class);
 					assertThat(context).hasSingleBean(DynamoDbTemplate.class);
 					assertThat(context).hasSingleBean(DynamoDbEnhancedClient.class);
 
-					ConfiguredAwsClient client = new ConfiguredAwsClient(context.getBean(DynamoDbClient.class));
-					assertThat(client.getEndpoint()).isEqualTo(URI.create("http://localhost:8090"));
-					assertThat(client.isEndpointOverridden()).isTrue();
+					ClusterDaxClient client = context.getBean(ClusterDaxClient.class);
+					software.amazon.dax.Configuration configuration = getConfiguration(client);
```

Comment:
To follow the same pattern as `ConfiguredAwsClient`, instead of `getConfiguration` method perhaps its better to have `ConfiguredDaxClient` class?